app/controllers/transition_defence_requests_controller.rb

```diff
@@ -34,6 +34,7 @@ def transition_params
       defence_request: defence_request,
       transition_to: requested_transition,
       user: current_user,
+      auth_token: session[:user_token]
```

Comment:
This is temporary as I'd like to put the `auth_token` on to the `current_user` object. The current user is created from the token anyway and we won't have to pass it around separately.

I thought about passing the client from the controller, but then none of the other transitions need it. So it seems unnecessary to instantiate it every time.